homeassistant/components/recorder/strings.json

```diff
@@ -7,5 +7,11 @@
       "database_engine": "Database Engine",
       "database_version": "Database Version"
     }
+  },
+  "issues": {
+    "maria_db_range_index_regression": {
+      "title": "Update MariaDB to {min_version} or later resolve a significant performance issue",
+      "description": "Older versions of MariaDB suffer from a significant performance regression when retrieving history data or purging the database. Update to version {min_version} or later and restart Home Assistant."
```

Comment:
Maybe add a little hint for add-on users?

"If you are using the MariaDB core add-on, make sure to update it to the latest version"?

Something like that?

../Frenck